webui/react/src/omnibar/actions.tsx

```diff
@@ -0,0 +1,44 @@
+import { message, Modal } from 'antd';
```

Comment:
**nit:** rename to `src/omnibar/actions/utils.ts`?

Reply:
I restructure this and nodes directory into `trees`